application/router.py

```diff
@@ -0,0 +1,106 @@
+from typing import Optional
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.orm import Session
+
+from application import schemas, crud, utils
+from dependencies import get_session
+
+
+router = APIRouter()
+
+
+@router.get("/cities/", response_model=list[schemas.CityInfo])
```

Comment:
The type annotation `list[schemas.CityInfo]` is not compatible with Python versions below 3.9. Consider using `List[schemas.CityInfo]` if you are using a version below 3.9.